packages/rule-components/src/ReportDetails/RiskOfChangeIcon.js

```diff
@@ -22,7 +22,10 @@ const RiskOfChangeIcon = ({ severity, label, ...props }) => {
             >
                 <SecurityIcon className="ins-risk-of-change-icon"/>
             </i>
-            { label && label.length > 0 && <span className='label'> { label } </span> }
+            {
+                label && label.length > 0 &&
+                <span className="ins-risk-of-change-label">{ label }</span>
```

Comment:
Can you change this class to `ins-c-risk-of-change__label` so we follow same pattern as with other classes?

Reply:
fixed